roles/static_inventory/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
What is the primary purpose of this role?  This part might need to stay in contrib?

Reply:
Ah, good point. The contrib playbooks let you use either a dynamic inventory (as in this PR) or use this role to generate a static inventory. That's not used in this PR, so I can drop it for now and we deal with this in another change.